docs/IntegrationTest.md

```diff
@@ -0,0 +1,191 @@
+    kubectl describe ingress $JENKINS_NAME
+    ```
+
+    Here's what that should look like (`#` can be any number, including hex digits):
+    
+    ```
+    $ user@cloudshell:~ (exampleproject)$ kubectl describe deployment $JENKINS_NAME
+    Name:               jenkins-test-host
+    Namespace:          default
+    CreationTimestamp:  ...
+    Labels:             chart=jenkins-0.32.9
+                        component=jenkins-test-host-jenkins-master
+                        heritage=Tiller
+                        release=jenkins-test-host
+    ...
+    ...
+    ...
+    
+    $ user@cloudshell:~ (exampleproject)$ kubectl describe ingress $JENKINS_NAME
+    Name:             jenkins-test-host
+    Namespace:        default
+    Address:          #.#.#.#
+    Default backend:  default-http-backend:80 (#.#.#.#:8080)
+    Rules:
+      Host                              Path  Backends
+      ----                              ----  --------
+      example.com
+                                           jenkins-test-host:8080 (<none>)
+    Annotations:
+      ingress.kubernetes.io/url-map:                k8s-um-default-jenkins-test-host--7f##############
+      kubernetes.io/ingress.allow-http:             false
+      kubernetes.io/ingress.global-static-ip-name:  jenkins-address
+      ingress.gcp.kubernetes.io/pre-shared-cert:    jenkins-host-certificate
+      ingress.kubernetes.io/backends:               {"k8s-be-#####--7f##############":"HEALTHY","k8s-be-#####--7f##############":"HEALTHY"}
+      ingress.kubernetes.io/https-forwarding-rule:  k8s-fws-default-jenkins-test-host--7f##############
+      ingress.kubernetes.io/https-target-proxy:     k8s-tps-default-jenkins-test-host--7f##############
+      ingress.kubernetes.io/ssl-cert:               jenkins-host-certificate
+    ```
+
+### Testing the plugin on Jenkins
+1. Make sure you have another cluster and service account set up for testing deployments as
+described in the GKE Plugin [usage documentation](Home.md#usage). You will need to repeat steps 3-8 
+for the [IAM Credentials](Home.md#iam-credentials) section on this new Jenkins instance. This is the
+reason that a certificate is needed, as you will need to ensure a secure encrypted connection when
+uploading the private key file to Jenkins.
+
+1. In your browser, go to `https://example.com` (replace with your domain) and you should see the
+Jenkins login page.
+
+1. Enter "admin" as the username, and enter the jenkins password retrieved above, then log in.
+
+1. By default, this jenkins master will have the latest release of the GKE Plugin. See [Source Build
+Installation](SourceBuildInstallation.md) for instructions on uploading a different version of the
+plugin.
+
+1. From the main jenkins page click **New Item**, then enter a name and choose **Freestyle project**.
+
+1. Under **Source Code Management**, select Git and enter this repository:
+https://github.com/craigatgoogle/testthrowaway.git
```

Comment:
We really shouldn't be including my one-off test repo in our official docs. We should find something more permanent to use here.

Reply:
How would you feel about including that file in with the other resources that I've included here? Alternatively we could link to something in the kubernetes documentation and have them copy it into their own file/repo

Reply:
Either option is works, up to you :)